src/plugins/kibana/public/management/styles/main.less

```diff
@@ -41,6 +41,30 @@ kbn-management-app {
 
 kbn-management-landing {
   padding: 5px;
+}
+
+  .management-panel-product {
+    margin-bottom: 10px;
+
+    .panel-body {
+      padding-bottom: 30px;
+    }
+  }
+
+  .management-section {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .management-link {
```

Comment:
I created this new class so we can scope the typography-related styles to the link text itself. These styles previously belonged to `panel-body`.